cpp/models/abm/person.cpp

```diff
@@ -189,4 +186,24 @@ bool Person::get_tested(const TestParameters& params)
         }
     }
 }
+
+uint32_t Person::get_person_id()
+{
+    return m_person_id;
+}
+
+void Person::set_cells(const std::vector<uint32_t>& cells)
```

Comment:
I think this is only used for testing? I think this should be removed, migrate_to does the same thing.